bbc/simorgh
# Apply UX styling to header

Applies CSS to the anchor element, adds an inner span to the anchor, and applies hover, focus and active styles. Due to the change in markup (additional inner span), we are re-implementing hover and focus. Adds a styled component svg.

* [x] Resolves #346, Resolves #354 

**Testing Instructions**
* Style: Ensure correct border bottom styles apply for each state (active, hover, focus); ensure tap area and svg logo agrees with visual design
* Keyboard only: Ensure anchor/logo is keyboard operable for each state (can be tabbed to, can be focused, can be activated)
* VoiceOver (iOS): on receiving focus, styling remains consistent at mobile viewport sizes
